### ISSUE TITLE
MiniZinc segmentation fault with optimization -O1 (MIP domains)

### ISSUE THREAD

**Issue by raphaelboudreault** [MiniZinc/libminizinc#524]
Hi,
I have a segmentation fault when running MiniZinc 2.5.5 (both Windows and Ubuntu bundled versions) on the following model with the optimization option `-O1` and with a value for the parameter `x` greater or equal to 10:
```
int: x;
set of int: X = 0..x;
var X: a;
array[X] of var bool: b;
array[X] of var X: c;

constraint forall(v in X)(
b[v] -> 
forall(w in X where w >= a)(c[v] <= w)
);
```
The fact that the bug depends on the value of `x` is a mystery to me. Thanks.

**Comment by Dekker1** [comment]
I looks like this issue has already been resolved on the `develop` branch and should be part of the next release.

**Comment by raphaelboudreault** [comment]
Thanks! Do you know which commit resolved the problem? I'm curious! Also, do you plan a release in a near future?

**Comment by Dekker1** [comment]
I'm afraid I do not know which commit solved the problem. There have been a lot of fixes and changes since 2.5.5. We hope to release version 2.6 of MiniZinc soon.
